style.css

```diff
@@ -60,6 +60,61 @@ header{
     background-color: #87b0ed;
 }
 
+/* Mute button */
+#mute-btn {
+    float: right;
+    padding-left: 5px;
+    padding-right: 5px;
+}
```

Comment:
instead of positioning the button just below the game, you can put it absolute at the bottom right corner

ex: on many site you must have seen a button  `go to top`